test/graph_test/base_graph_test.cpp

```diff
@@ -96,15 +97,26 @@ void BaseGraphTest::createDBAndConn() {
 }
 
 void BaseGraphTest::initGraph(const std::string& datasetDir) const {
-    if (conn) { // normal conn
-        TestHelper::executeScript(datasetDir + TestHelper::SCHEMA_FILE_NAME, *conn);
-        TestHelper::executeScript(datasetDir + TestHelper::COPY_FILE_NAME, *conn);
+    // if conn is null we need to use the connMap
+    // as per the conn documentation
```

Comment:
Comments should start with a capital letter and end with a `.` Also use the full span before wrapping.